core/test/functional/routes/api/upload_spec.js

```diff
@@ -128,5 +144,20 @@ describe('Upload API', function () {
                     done();
                 });
         });
+
+        it('import should fail if profile image is not square', function (done) {
```

Comment:
I think this can be tested in a unit test? We rly try to avoid routing tests. Let me know what you think

Reply:
Discussed: we leave it like this for now 👍